### PR TITLE
Add extra setup instructions for postgres

### PR DESCRIPTION
**Pull request by asross**
Ran into some very minor postgres problems setting up the repo, hopefully this is a helpful fix!

### CODE REVIEW

File: sunrise/settings/common.py [hunk 1]
```diff
@@ -77,7 +77,7 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'sunrise_strike_circles',
-        'USER': 'jesse',
+        'USER': env('POSTGRES_DB_NAME'),
```
Comment (jlevers) [inline sunrise/settings/common.py]: Just to clarify here, the `USER` key is actually the Postgres *user*name, not the Postgres *database* name. I'm also going to add the Postgres user's password to the `.env` file. Thanks for helping out this, it really needed to be changed.

**Comment by jlevers** [comment]
Thanks @asross! I made a comment above about a few little changes to be made -- when complete, this will fix #15.

File: sunrise/settings/common.py (hunk 1 above)
Comment (asross) [reply sunrise/settings/common.py]: Sounds good, will do!

**Comment by asross** [comment]
@jlevers updated the PR! Also created an environment variable for POSTGRES_HOST since it seems like that could be different in dev vs. production.

**Comment by jlevers** [comment]
Looks great, thank you! Merged, and resolved #15.

**Pull request merged**